spec/support/boxes/line_item_box.cr

```diff
@@ -0,0 +1,8 @@
+require "uuid"
```

Comment:
I'd consider requiring this in `src/lucky_record.cr` so it is always available and you don't need to manually require it in other files